apps/chat-e2e/src/tests/messageTemplate.test.ts

```diff
@@ -846,7 +869,436 @@ dialTest(
+        await additionalShareUserConversationDropdownMenu.selectMenuOption(
+          MenuOptions.replay,
+        );
+        await additionalShareUserDialHomePage.mockChatTextResponse(
+          MockedChatApiResponseBodies.simpleTextBody,
+        );
+        await additionalShareUserChat.startReplay(
+          simpleConversationMessage.messages[0].content,
+          true,
+        );
+        await additionalShareUserVariableModalAssertion.assertElementState(
+          additionalShareUserVariableModalDialog,
+          'visible',
+        );
+        await additionalShareUserVariableModalAssertion.assertPromptDescription(
+          expectedPreviewContent,
+        );
+      },
+    );
+  },
+);
+
+dialAdminTest(
+  `Replay with Message template: by the administrator from 'Approved required'.\n` +
+    `Replay with Message template: by the user from 'Organization'`,
+  async ({
+    conversationData,
+    dataInjector,
+    publicationApiHelper,
+    adminPublicationApiHelper,
+    publishRequestBuilder,
+    dialHomePage,
+    organizationConversations,
+    conversationDropdownMenu,
+    chat,
+    variableModalDialog,
+    variableModalAssertion,
+    adminDialHomePage,
+    adminChat,
+    adminApproveRequiredConversations,
+    adminApproveRequiredConversationDropdownMenu,
+    adminVariableModal,
+    adminVariableModalAssertion,
+    setTestIds,
+  }) => {
+    setTestIds('EPMRTC-4281', 'EPMRTC-4282');
+    let simpleConversationMessage: Conversation;
+    let templateConversationMessage: Conversation;
+    let conversation: Conversation;
+    const publicationRequestName = GeneratorUtil.randomPublicationRequestName();
+    let publication: Publication;
+
+    await dialTest.step(
+      'Prepare conversation with 2 requests where one of them is based on message template',
+      async () => {
+        simpleConversationMessage =
+          conversationData.prepareDefaultConversation();
+        conversationData.resetData();
+        templateConversationMessage =
+          conversationData.prepareConversationBasedOnMessageTemplate(
+            requestContent,
+            rowsMap,
+          );
+        conversation = conversationData.prepareHistoryConversation(
+          simpleConversationMessage,
+          templateConversationMessage,
+        );
+        await dataInjector.createConversations([conversation]);
+      },
+    );
+
+    await dialTest.step(
+      'Create publication request for the conversation',
+      async () => {
+        const publishRequest = publishRequestBuilder
+          .withName(publicationRequestName)
+          .withConversationResource(conversation, PublishActions.ADD)
+          .build();
+        publication =
+          await publicationApiHelper.createPublishRequest(publishRequest);
+        publicationsToUnpublish.push(publication);
+      },
+    );
+
+    await dialTest.step(
+      'Create a replay conversation from "Approve required" section for publication request conversation',
```

Comment:
I can barely imagine the reason for that functionality. Are we sure that it is needed?

Reply:
Manual QA requested automation of this feature)

Reply:
I believe the goal is to check whether shared/published conversations can also be parametrized